src/Contexts/section.actions.jsx

```diff
@@ -266,7 +183,8 @@ export const saveSections = () => async ({dispatch, state}) => {
 			'patch',
 			`lesson/${lesson._id}/sections`,
 			section._id,
-			changes
+			changes,
```

Comment:
```
hashes[index] = generateHash(section);
return editorWS.emit(
			'patch',
			`lesson/${lesson._id}/sections`,
			section._id,
			changes,
			{ state: 'diff' }
		)
```

Reply:
Können wir den input für das emit zu einen object umbiegen? Direkt so 5 parameter reingeben finde ich zu viel. Zudem ist die query bei lesson an 4. Stelle, hier ist sie an 5. Das mag für die Sockets erst mal keine Rolle spielen, aber der Emfänger muss dann auch wissen, was es damit auf sich hat.

Reply:
die funktion gibt es so eins zu eins and den socket emit weiter, dass ist die von socket bzw von socket feathers vorgegeben schreibweise

Reply:
Ok dann geht feathers über die Methode und mapped, Methode, Servicesname, id, data und query bzw. function to execute. Ließt sich trotzdem nicht schön. Ein setzen über Object keys und späteres mapping wäre intuitiver.